# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: servo/components/style/values/specified/box.rs
# Commit: 2d16e9e90401
# Full Hash: 2d16e9e904010ba3389aad9729bac4f3744a588d
# Author: Mats Palmgren <mats@mozilla.com>
# Date: 2021-01-30 05:43:48
# Regressor Bug: 1683748
# File Overlap Count: 3
# Description:
#   Bug 1683748 - Support Grid/Flex/Table/Column layout for the rendered legend of a fieldset.  r=emilio
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D101555
# ==============================================================================

diff -r 5b99d9a7ad8d -r 2d16e9e90401 servo/components/style/values/specified/box.rs
--- a/servo/components/style/values/specified/box.rs	Fri Jan 29 11:30:44 2021 +0000
+++ b/servo/components/style/values/specified/box.rs	Fri Jan 29 15:45:42 2021 +0000
@@ -265,6 +265,12 @@
         .unwrap()
     }
 
+    /// Returns the raw underlying u16 value.
+    #[inline]
+    pub const fn to_u16(&self) -> u16 {
+        self.0
+    }
+
     /// Whether this is `display: inline` (or `inline list-item`).
     #[inline]
     pub fn is_inline_flow(&self) -> bool {